types/v56/types.go

```diff
@@ -3357,6 +3358,27 @@ type QueryResultTaskRecordType struct {
 	Metadata         *Metadata `xml:"Metadata,omitempty"`
 }
 
+// QueryResultOrgVdcRecordType represents an Organisation record
+type QueryResultOrgRecordType struct {
+	HREF               string    `xml:"href,attr,omitempty"`
+	Type               string    `xml:"type,attr,omitempty"`
+	ID                 string    `xml:"id,attr,omitempty"`
+	Name               string    `xml:"name,attr"`
+	DisplayName        string    `xml:"displayName,attr,omitempty"`
+	IsEnabled          bool      `xml:"IsEnabled,attr,omitempty"`
```

Comment:
The name of the field is `isEnabled`, starting with lowercase 
```
<OrgRecord 
name="xxxxxx" 
isReadOnly="false" 
isEnabled="true" 
deployedVMQuota="50" 
storedVMQuota="50" 
displayName="xxxxxxx" 
canPublishCatalogs="true" 
numberOfCatalogs="3" 
numberOfVdcs="5" 
numberOfVApps="2" 
numberOfGroups="1" 
numberOfDisks="0" 
canPublishExternally="true" 
href="https://notarealurl.com/api/org/4f4aac24-dc39-4d76-8b43-edb0eeb2f2ec" 
numberOfRunningVMs="2"/>
```

Reply:
will be fixed in next commit